pkg/controller/backup_controller.go

```diff
@@ -143,6 +150,21 @@ func NewBackupController(
 				switch backup.Status.Phase {
 				case "", velerov1api.BackupPhaseNew:
 					// only process new backups
```

Comment:
This comment should be updated.
Not only new backups are handle, since this change.

Reply:
Sure, I will update that.